app/src/main/res/layout-w720dp/layout_search.xml

```diff
@@ -101,23 +101,47 @@
 
         </HorizontalScrollView>
 
-        <TextView
-            android:id="@+id/searchDeepSearchTV"
+        <LinearLayout
+            android:id="@+id/deepSearchContainer"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
```

Comment:
can you make the whole container fill width, and align both `notFindingTextView` & `tryDeepSearchButton` at both ends where the remaining space is filled. i.e, to right-align `tryDeepSearchButton`.

Reply:
Thanks for the suggestion! I've updated the layout so that the container fills the full width, and aligned `notFindingTextView` and `tryDeepSearchButton` to opposite ends with the button now right-aligned and the space between them properly distributed.

![Screenshot 2025-04-22 082629](https://github.com/user-attachments/assets/69c664c3-65c0-48af-9684-acceed982f37)

Please let me know if anything else needs to be changed.